src/jmclient/wallet.py

```diff
@@ -2046,8 +2049,8 @@ class BIP39WalletMixin(object):
     _BIP39_EXTENSION_KEY = b'seed_extension'
     MNEMONIC_LANG = 'english'
 
-    def _load_storage(self):
-        super()._load_storage()
+    def _load_storage(self, load_cache=True):
```

Comment:
```suggestion
    def _load_storage(self, load_cache: bool = True) -> None:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1616/commits/ef1d76e9db09170db94ac0c10efe7b73866b64e3